src/com/sproutlife/model/GameModel.java

```diff
@@ -15,45 +15,49 @@
 import com.sproutlife.model.step.GameStep;
 import com.sproutlife.model.step.GameStepListener;
 
+/**
+ * This is a class used to manage the game model essentials.
+ *
+ * @author Alex Shapiro
+ */
 public class GameModel {
 
-    Echosystem echosystem;  
+    private Echosystem echosystem;
 
-    GameClock clock;
+    private GameClock clock;
 
-    GameStep gameStep;
+    private GameStep gameStep;
 
-    GameThread gameThread;
+    private GameThread gameThread;
 
-    Settings settings;
+    private Settings settings;
 
-    Stats stats;   
-    
+    private Stats stats;
+
+    /**
+     * GameModel Constructor
+     */
     public GameModel(Settings settings, ReentrantReadWriteLock interactionLock) {
         this.settings = settings;
         this.clock = new GameClock();
         echosystem = new Echosystem(clock);
-        gameStep = new GameStep(this);       
+        gameStep = new GameStep(this);
         gameThread = new GameThread(this, interactionLock);
-        stats = new Stats(this);
+		stats = new Stats(this);
```

Comment:
@DanyYanev there are still tabs at the start of this line, and in other places below.